docs/cloud/azureai/create-run-with-compute-session.md

```diff
@@ -1,16 +1,15 @@
-# Create run with automatic runtime
+# Create run with compute session
```

Comment:
Please check the work item, we also need to do some reorg of the pages. Let's sync tommorow

Reply:
OK